src/style.css

```diff
@@ -1,3 +1,85 @@
+:root {
+  --main-color: #000;
+  --hover-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, you could specify a generic family at the end like this: `font-family: Roboto, Arial, Helvetica, sans-serif, sans-serif;`